packages/base-driver/lib/protocol/routes.js

```diff
@@ -250,21 +250,6 @@ export const METHOD_MAP = /** @type {const} */ ({
   //
   // Appium specific
   //
-  '/session/:sessionId/ime/available_engines': {
-    GET: {command: 'availableIMEEngines'},
-  },
-  '/session/:sessionId/ime/active_engine': {
-    GET: {command: 'getActiveIMEEngine'},
-  },
-  '/session/:sessionId/ime/activated': {
-    GET: {command: 'isIMEActivated'},
-  },
-  '/session/:sessionId/ime/deactivate': {
-    POST: {command: 'deactivateIMEEngine'},
-  },
-  '/session/:sessionId/ime/activate': {
-    POST: {command: 'activateIMEEngine', payloadParams: {required: ['engine']}},
-  },
```

Comment:
For this file, I'd like to keep these (for now) and put `deprecated` first before removing them like other removals because we haven't marked as `deprecated` yet in current `master`. Instead, I'm good at removing these method from types to imply we don't expect to be implemented these endpoints

Reply:
restored